Scrape test fails on CPython 3.14

As observed in the 3.14 CI run on `main`: https://github.com/fox-it/dissect.target/actions/runs/27349456919/job/80807156411

```
[808](https://github.com/fox-it/dissect.target/actions/runs/27349456919/job/80807156411#step:4:6814)
=================================== FAILURES ===================================
__________________________________ test_find ___________________________________
[gw0] linux -- Python 3.14.5 /home/runner/work/dissect.target/dissect.target/.tox/3.14/bin/python

target_bare = <Target /home/runner/work/dissect.target/dissect.target/.tox/3.14/tmp/popen-gw0/test_find0/MockTarget-ref4uz60>

    def test_find(target_bare: Target) -> None:
        target_bare.add_plugin(ScrapePlugin)
    
        buf = (b"\x00" * 1024 * 64) + (b"ABCD" + b"\x00" * ((1024 * 4) - 4)) + (b"\x00" * 1024 * 60)
        mock_disk = RawContainer(io.BytesIO(buf))
        target_bare.disks.add(mock_disk)
    
        mock_progress = Mock()
        for disk, stream, needle, offset, match in target_bare.scrape.find(b"ABCD", progress=mock_progress):
            assert disk.size == 1024 * 128
            assert stream.size == 1024 * 128
            assert needle == b"ABCD"
            assert offset == 1024 * 64
            assert not match  # only for regex matches
    
        for i in range(0, 1024 * 128, 8192):
>           mock_progress.assert_any_call(mock_disk, i, 1024 * 128)

tests/plugins/scrape/test_scrape.py:142: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Mock id='140160715176128'>
args = (<Container type=raw size=131072 vs=None>, 8192, 131072), kwargs = {}
expected = call(<Container type=raw size=131072 vs=None>, 8192, 131072)
cause = None
actual = [call(<Container type=raw size=131072 vs=None>, 0, 131072)]
expected_string = 'mock(<Container type=raw size=131072 vs=None>, 8192, 131072)'

    def assert_any_call(self, /, *args, **kwargs):
        """assert the mock has been called with the specified arguments.
    
        The assert passes if the mock has *ever* been called, unlike
        `assert_called_with` and `assert_called_once_with` that only pass if
        the call is the most recent one."""
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        cause = expected if isinstance(expected, Exception) else None
        actual = [self._call_matcher(c) for c in self.call_args_list]
        if cause or expected not in _AnyComparer(actual):
            expected_string = self._format_mock_call_signature(args, kwargs)
>           raise AssertionError(
                '%s call not found' % expected_string
            ) from cause
E           AssertionError: mock(<Container type=raw size=131072 vs=None>, 8192, 131072) call not found

/opt/hostedtoolcache/Python/3.14.5/x64/lib/python3.14/unittest/mock.py:1057: AssertionError
```

--- a/tests/plugins/scrape/test_scrape.py
+++ b/tests/plugins/scrape/test_scrape.py
@@ -138,5 +138,5 @@ def test_find(target_bare: Target) -> None:
         assert offset == 1024 * 64
         assert not match  # only for regex matches
 
-    for i in range(0, 1024 * 128, 8192):
+    for i in range(0, 1024 * 128, io.DEFAULT_BUFFER_SIZE):
         mock_progress.assert_any_call(mock_disk, i, 1024 * 128)

